include/aws/http/connection.h

```diff
@@ -241,6 +275,28 @@ struct aws_http_client_connection_options {
      * reaches 0, no further data will be received.
      **/
     bool manual_window_management;
+
+    /**
+     * HTTP/2 specific options.
+     * Optional for HTTP/2 connection. Ignored for HTTP/1 connection.
+     */
+    struct aws_http2_connection_options {
```

Comment:
If we support HTTP/2 server in the future, this structure may need to be moved?

Reply:
we'll worry about it in the future
server.h could just include connection.h